tests/conftest.py

```diff
@@ -18,6 +18,9 @@ def spark_context(request):
     return sc
 
 
-def row_to_dict(row):
+@pytest.fixture(scope="session")
+def row_to_dict():
     """Convert pyspark.Row to dict for easier unordered comparison"""
-    return {key: row[key] for key in row.__fields__}
+    def func(row):
+        return {key: row[key] for key in row.__fields__}
+    return func
```

Comment:
This is a bit odd at first, but something that is common in pytest: a fixture that returns a function, so it can be shared by many tests and is centrally implemented.